nick-fury/check_availability.py

```diff
@@ -0,0 +1,27 @@
+import sys
+import MySQLdb
+
+
+con1 = MySQLdb.connect(host='localhost',

```

Comment:
Its recommended to store DB secrets as environment variables

Reply:
I have resolved this issue.